oshi-core/src/test/java/oshi/util/TupleTest.java

```diff
@@ -47,23 +47,22 @@ public void testTuples() {
         Quintet<String, Integer, Long, Character, BigInteger> quintet = new Quintet<>("D", 4, Long.valueOf("0"), 'd',
                 BigInteger.ZERO);
 
-        assertEquals("A", pair.getA());
-        assertEquals("B", triplet.getA());
-        assertEquals("C", quartet.getA());
-        assertEquals("D", quintet.getA());
+        assertEquals(pair.getA()+" should be A","A", pair.getA());
```

Comment:
This should be `assertEquals("pair.getA() should be A", "A", pair.getA());` and similar for the others.

Reply:
Sorry for my wrong understanding. I have updated as suggested .